website/content/docs/templates/json_to_hcl.mdx

```diff
@@ -0,0 +1,157 @@
+---
+description: |
+  This section highlights the differences between JSON and HCL2 templates, and explains the parallels between the two specifications.
+page_title: JSON to HCL
+---
+
+# Packer templates
+
+Packer uses templates to orchestrate builds for one or multiple images.
+In JSON, you would declare a series of builders, provisioners and post-processors to build images.
+In HCL2, things are different, as the configuration language allows you to specify builders
+through sources, and weave them in build blocks.
+
+This document aims to explain the parallels between the two configuration files, and what you
+should expect when moving a template away from JSON to HCL2.
```

Comment:
What do you think about adding a blurb about the `hcl2_upgrade` command for converting JSON to HCL2 templates and linking out the Upgrade tutorial https://developer.hashicorp.com/packer/tutorials/configuration-language/hcl2-upgrade?